rotkehlchen/globaldb/cache.py

```diff
@@ -89,8 +105,13 @@ def globaldb_get_general_cache_last_queried_ts_by_key(
     element returns Timestamp(0)
     """
     cache_key = _compute_cache_key(key_parts)
+    unique_cache = any(cache_key.startswith(key_part) for key_part in UNIQUE_CACHE_KEYS)
+    if unique_cache:
+        cache = UNIQUE_CACHE
+    else:
+        cache = GENERAL_CACHE
```

Comment:
why not add this logic to `_compute_cache_key` and avoid repeating it in each function? I'm thinking in changing the return type to `cache_key, cache_type` and avoid repeating this in every function that uses it